content/docs/command-reference/exp/run.md

```diff
@@ -63,11 +71,15 @@ committing them to the Git repo. Unnecessary ones can be [cleared] with
 > In addition to the following, `dvc exp run` accepts the options in `dvc repro`
 > except for `--glob`, `--no-commit`, and `--no-run-cache`.
 
-- `-S [<filename>:]<param_name>=<param_value>`,
-  `--set-param [<filename>:]<param_name>=<param_value>` - set the value of
```

Comment:
What do you think about keeping the basic `<param_name>=<param_value>` syntax in the description somewhere?